src/Polyndrome_Numbers/Polyndrome.java

```diff
@@ -1 +1,35 @@
-//
\ No newline at end of file
+package Polyndrome_Numbers;
+
+import java.util.Scanner;
+
+public class Polyndrome {
+
+    public static boolean polyndrome(int n){
+
+        if (n < 0) return false;
```

Comment:
also consider n%10==0 which we discussed in class

Reply:
it's considered on while (n!==0)